### PR TITLE
Identity test scripts

### PR DESCRIPTION
**Pull request by robmarch2**

### CODE REVIEW

File: identity/test-scripts/change-email.feature [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    Then the user will be informed they must enter a valid email address
+
+  @manual
+  Scenario: WS-31-004b: Change email with correct credentials after previous incorrect attempt
+    Given the user enters a invalid email address
+    And the user enters a incorrect password to confirm
+    When the user selects ‘update email’
+    And the user enters a valid email address
+    And the user selects ‘update email’
+    Then ……  Bug raised as not behaving as expected WS-186 and I don’t think captured in my previous testing or bugs
```
Comment (robmarch2) [inline identity/test-scripts/change-email.feature:46]: TODO: need the correct assertion here

File: identity/test-scripts/change-email.feature [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+  @manual
+  Scenario: WS-31-005: Change email with email in use and invalid password
+    Given the user enters a valid email address but it is already in use
+    And the user enters a invalid password to confirm
+    When the user selects ‘update email’
+    Then the modal refreshes and the original email address is displayed with no error message
+    And … Think this is addressed by WS-176 which is at Peer Review
```
Comment (robmarch2) [inline identity/test-scripts/change-email.feature:54]: TODO: need the correct assertion here

File: identity/test-scripts/change-email.feature [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+  @manual
+  Scenario: WS-31-006: Change email address with correct email but invalid password
+    Given the user enters a valid email address that is not already in use
+    And the user enters a incorrect password to confirm
+    When the user selects ‘update email’
+    Then … There was a bug WS-175 which is in Peer Review
```
Comment (robmarch2) [inline identity/test-scripts/change-email.feature:61]: TODO: need the correct assertion here

File: identity/test-scripts/change-password.feature [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    And the user selects Change Password
+    Then the Change Password Screen opens with the fields pre populated
+
+  @manual
+  Scenario: WS-31-012c: Change passwords correctly but do not update
+    Given the user populates all the fields correctly
+    And the user clicks the close on the box without updating the password
+    And the user selects Change Password
+    When the user selects Update Password without changing any fields
+    Then the user is returned to the My Account Screen
```
Comment (robmarch2) [inline identity/test-scripts/change-password.feature:80]: TODO: is this the correct assertion here?

File: identity/test-scripts/request-delete.feature [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    Given the User changes their mind and does not want to delete
+    When the user click ‘No, take me back to my account’
+    Then the request deletion modal is closed
+
+  @manual
+  Scenario: WS-33-002: Request delete but enter an invalid password
+    Given the user has the 'Request deletion' screen open
+    And the User enters an invalid password
+    When the user click ‘Yes, delete my account’
+    Then the user is informed 'Incorrect password.' and the field is highlighted
```
Comment (robmarch2) [inline identity/test-scripts/request-delete.feature:20]: TODO: is this the correct assertion here?

File: identity/test-scripts/request-delete.feature [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+  Background:
+    Given a library user has logged in successfully and navigated to the My Account screen
+    And the user clicks the 'Request deletion' option from the ‘My Account’ screen
+
+  @manual
+  Scenario: WS-33-001: Cancel out of request to delete
+    Given the User changes their mind and does not want to delete
+    When the user click ‘No, take me back to my account’
+    Then the request deletion modal is closed
```
Comment (robmarch2) [inline identity/test-scripts/request-delete.feature:13]: TODO: is this the correct assertion here?

File: identity/test-scripts/request-delete.feature [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    When the user click ‘Yes, delete my account’
+    And the user closes the box and does not select an action
+    And the user selects ‘Request deletion’
+    Then the Screen opens with the highlighted field and ‘Enter your current password’ message appearing (Bug raised ws-188)
+
+  @manual
+  Scenario:WS-33-005: Request delete with correct password and action
+    Given the User enters an valid password
+    When the user click ‘Yes, delete my account’
+    Then the user .. WS-179 Bug - In Progress
```
Comment (robmarch2) [inline identity/test-scripts/request-delete.feature:53]: TODO: need the correct assertion here

**Comment by kenoir** [comment]
Closed as out-of-date.

**Pull request closed without merging**